.github/ISSUE_TEMPLATE/4_documentation_change.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Documentation change request
+about: Report documentation issue or request documentation changes
+title: ''
+labels: documentation
+assignees: ''
+
+---
+
+**Summary**
+Include a short summary of the issue or request. 
+See the sections below
+for factors important for a documentation
+issue.
+
+**URLs**
+Include pointers to pages that are impacted.
+
+**Additional details**
+Provide a detailed description of the expected changes in documentation
```

Comment:
Minor update to the Additional Details description: please change to: "Provide a detailed description of the expected changes in the documentation and any suggestions you have."

Reply:
Done.